### DISCUSSION TITLE
[M-Z] (10 Febbraio 2020) Esercizio 7

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#464] [Esami & Esoneri M-Z]
## Traccia

<img width="1065" alt="image" src="https://github.com/user-attachments/assets/e7317c87-f813-41e8-952b-be3fdada5cb0">

**Comment by matypist** [comment]
Segue una proposta di soluzione di Andrea Princic (@Princic-1837592), tratta dal suo file di soluzioni [20200210.pdf](https://raw.githubusercontent.com/Princic-1837592/metodi-matematici/refs/heads/main/soluzioni/20200210.pdf) nel suo [repository GitHub](https://github.com/Princic-1837592/metodi-matematici) contenente le sue soluzioni agli esami del prof. Piperno:

![image](https://github.com/user-attachments/assets/fc23734a-c80e-4814-8718-bad3d0aca2d5)
